docs/architecture.md

```diff
@@ -85,7 +85,7 @@ This section provides an overview on folders / sub-projects that exist in this r
   - `/env`: `.env` files that contain environment variables for different configuration of zkSync Server / Prover.
   - `/js`: Configuration files for JavaScript applications (such as Explorer).
   - `/tokens`: Configuration of supported Ethereum ERC-20 tokens.
-- `/infrastructure`: Application that aren't naturally a part of zkSync core, but are related to it.
+- `/infrastructure`: Applications that aren't naturally a part of zkSync core, but are related to it.
```

Comment:
`Application` can be both countable and uncountable.

Reply:
@0xb10ckdev Ohh sorry I didn't know, I reverted this.